scripts/pr-previews/builder.py

```diff
@@ -130,14 +130,14 @@ def _copy_local_content(root_dir: Path) -> None:
             if dir.startswith("public")
             else root_dir / dir
         )
-        shutil.copytree(dir, dest)
+        shutil.copytree(dir, _add_locale_to_docs(dest))
```

Comment:
You should instead call this on line 131 so that the expected type for add_locale is `str`. Note that in 140 you are passing it a string, which makes sense.

This way you also avoid the awkward `Path -> str -> Path` conversion